src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkAuctionResults.tsx

```diff
@@ -0,0 +1,134 @@
+import { Box, Column, Flex, Join, Message, Spacer, Text } from "@artsy/palette"
+import { useEffect, useState } from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { LoadingArea } from "Components/LoadingArea"
+import createLogger from "Utils/logger"
+import { MetaTags } from "Components/MetaTags"
+import { ArtistAuctionResultItemFragmentContainer } from "Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem"
+import { extractNodes } from "Utils/extractNodes"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
+import { MyCollectionArtworkAuctionResults_artist } from "__generated__/MyCollectionArtworkAuctionResults_artist.graphql"
+import { RouterLink } from "System/Router/RouterLink"
+
+const logger = createLogger("MyColectionArtworkAuctionResults.tsx")
+
+interface MyColectionArtworkAuctionResultsProps {
+  relay: RelayRefetchProp
+  artist: MyCollectionArtworkAuctionResults_artist
+}
+
+const AuctionResultsContainer: React.FC<MyColectionArtworkAuctionResultsProps> = ({
+  artist,
+  relay,
+}) => {
+  const { slug, internalID, name, auctionResultsConnection } = artist
+
+  const results = extractNodes(auctionResultsConnection)
+
+  const [isLoading, setIsLoading] = useState(false)
+
+  useEffect(() => {
+    setIsLoading(true)
+
+    const relayRefetchParams = {
+      artistID: slug,
+      artistInternalID: internalID,
+    }
+
+    relay.refetch(relayRefetchParams, null, error => {
```

Comment:
Curious why are we refetching on mount (via `useEffect`)? It seems like this should only execute based on a user action. Since the page is server-side-rendered, the results of this fetch should already be in place.

Reply:
Right, we don't need it. Removed the `UseEffect`
And we also don't need `createRefetchContainer`,  using `createFragmentContainer` instead